schema.graphql

```diff
@@ -0,0 +1,60 @@
+"""
```

Comment:
are the contents of this file autogenerated?

Reply:
Yes they are using the task `rails graphql:schema:dump`